### ISSUE TITLE
Expected Lua table argument in nvim_set_hl call

### ISSUE THREAD

**Issue by Olical** [marko-cerovac/material.nvim#101]
Hi there, I updated the plugin this morning and I started seeing the following as I boot Neovim:

```viml
Error executing vim.schedule lua callback: ...te/pack/packer/start/material.nvim/lua/material/util.lua:77: Expected lua table
stack traceback:
        [C]: in function 'nvim_set_hl'
        ...te/pack/packer/start/material.nvim/lua/material/util.lua:77: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
```

I have a feeling this is either from a recent change (which you'll probably spot quickly) _or_ it's to do with Neovim's breaking API changes. I'm on Neovim v0.7.2, Arch Linux.

I don't see anything relevant in the scary thread though: https://github.com/neovim/neovim/issues/14090#issuecomment-1170955189

My config is as follows (sorry for the Fennel):

```fennel
(material.setup
  {:custom_highlights {:FloatBorder "#1A1A1A"}
   :borders true
   :high_visibility {:darker true}})
```

Not sure if there was a breaking config change? I can't spot anything obvious. My apologies if it's due to my config!

**Comment by marko-cerovac** [comment]
I don't know much about fennel. As far as I can tell, the lua equivalent would be:
```lua
require("material").setup({
    custom_highlights = {
        FloatBorder = "#1A1A1A"
    }
})
```
This woul'd be the wrong way to do this because the color isn't a string, but a table.
Please try this:
```lua
require("material").setup({
    custom_highlights = {
        FloatBorder = {
            fg = "#A1A1A1"
        }
    }
})
```

**Comment by Olical** [comment]
Ah that's all I needed, yep! I wonder why this has worked fine up until today... 🤔

**Comment by matthias-Q** [comment]
I have the same issue. The example in the Readme.md states that
`CursorLine = { fg = '#0000FF', gui = 'underline' },`
should work. However, the key 'gui' is not defined.

In my config it fails in the following line:
`TSFunction = { fg = "#82AAFF", style = "bold,italic" },`

It worked a few commits ago. Can you check what the problem is? I am not using fennel.

**Comment by marko-cerovac** [comment]
@matthias-Q Thanks for reminding me to fix the README. The reason behind all this is that I changed the way the theme sets highlights. Before, it used the vim-script ```:highlight``` command, but now it uses a lua function which makes the theme a bit quicker. The issue is that the lua function and the ```:highlight``` command use different arguments. I've now updated the README to show the new and correct way of doing this. In you case, that would be:
```lua
TSFunction = {
    fg = "#82AAFF",
    bold = true,
    italic = true
}
```
